### DISCUSSION TITLE
Are there any plan of multiple language support on machinaris web UI?

### DISCUSSION THREAD

**Discussion by jimuyouyou** [guydavis/machinaris#352] [Q&A] [answered]
It would be a great gift for every chia farmer in the world as as whole, if this is listed in the dev plan.

**Comment by guydavis** [comment, marked as answer]
Great suggestion! I have created issue #353 for this request.  Please add details of the language support you are looking for.  

I have done translations in other software before for Latin languages such as French and Spanish.  I don't experience with translating for Asian locales.  Please share tips on the best approaches.  Thanks!
